Sources/MapboxMaps/Ornaments/OrnamentsManager.swift

```diff
@@ -2,10 +2,19 @@ import UIKit
 
 public enum OrnamentPosition: String, Equatable {
     // Clockwise from top left
+    @available(*, deprecated, renamed: "topLeading")
     case topLeft
+    @available(*, deprecated, renamed: "topTrailing")
     case topRight
+    @available(*, deprecated, renamed: "bottomLeading")
     case bottomRight
+    @available(*, deprecated, renamed: "bottomTrailing")
```

Comment:
@ZiZasaurus, we've noticed this change during latest Maps SDK integration. Just want to make sure that this is correct naming. Shouldn't it be `bottomLeading` for `bottomLeft` and `bottomTrailing` for `bottomRight`, similarly to top positions?

Reply:
Yeah, can confirm it's a bug, bottomTrailing is actually shows on the left side of the screen😂

Reply:
Imho we shouldn't deprecate left/right options, UIKit has both leftAnchor and leadingAnchor, not all people change the layout of the whole app to support RTL, so we should support all options.

Reply:
Thanks for this! Pushing this change for the next release but for now we will be keeping the deprecation as is.

Reply:
@mapbox/maps-ios Should we revert it then?

Reply:
Seems to be fixed in here https://github.com/mapbox/mapbox-maps-ios/pull/1563

Reply:
@evil159 Sorry, forgot to mention this one:

> Imho we shouldn't deprecate left/right options, UIKit has both leftAnchor and leadingAnchor, not all people change the layout of the whole app to support RTL, so we should support all options.

Reply:
👍  -- I like having both naming options